src/Umbraco.Web/Editors/PasswordChanger.cs

```diff
@@ -181,9 +181,17 @@ public Attempt<PasswordChangedModel> ChangePasswordWithMembershipProvider(string
                 //ok, we should be able to reset it
                 try
                 {
-                    var newPass = membershipProvider.ResetPassword(
-                        username,
-                        membershipProvider.RequiresQuestionAndAnswer ? passwordModel.Answer : null);
+                    string newPass = passwordModel.NewPassword;
```

Comment:
The NewPassword logic is sort of handled below https://github.com/umbraco/Umbraco-CMS/pull/3528/files?w=1#diff-8529397cc05f7165e5300ea16e775622R209 and we also need to do a check on the `membershipProvider.ChangePassword` like below to ensure that this membership provider is able to manually change the password.

The reason why `passwordModel.Reset` is true and the NewPassword is set, is because that's how we instruct either identity or the membership provider that this is not the user of the password that is changing the password. Otherwise if it's the user themselves changing the password, they would supply their existing password and the `passwordModel.Reset` would be false. 

I'll update some of this code and add some more code comments to it